src/client/theme-default/components/VPHero.vue

```diff
@@ -16,6 +17,22 @@ defineProps<{
   image?: DefaultTheme.ThemeableImage
   actions?: HeroAction[]
 }>()
+const loaded = (e: Event) => {
+  const img = e.target as HTMLImageElement
+  const colors = getColors(img)
+  if (colors) {
+    const root = document.documentElement
+    let v: string[] = []
+    const useColors = colors.slice(0, 5)
+    const gap = Math.floor(100 / (useColors.length - 1))
+    useColors.forEach((c: number[], i: number) => {
+      v.push(`rgb(${c[0]}, ${c[1]}, ${c[2]}) ${i * gap}%`)
+    })
+    const bg = `linear-gradient(-45deg, ${v.join(', ')})`
+    root.style.setProperty('--vp-home-hero-image-background-image', bg)
+    root.style.setProperty('--vp-home-hero-image-filter', 'blur(72px)')
```

Comment:
I refer to the vite official website for the value here, and set it directly to `blur(72px)`. Maybe a configuration should be provided for users to set by themselves?